src/components/DRB/DRBCard.test.tsx

```diff
@@ -73,6 +73,6 @@ describe("DRBCard", () => {
     const titleLink = screen.getByText(/Spanish Title/i)
 
     expect(titleLink).toBeInTheDocument()
-    expect(titleLink).toHaveAttribute("lang", "es")
```

Comment:
I modified the DRB title component to use a CardHeading with a url prop instead of rendering the Link as a child. This puts the lang attribute on the surrounding h3 which is why I changed this to check the parentElement.